packages/shared/src/graphql/sources.ts

```diff
@@ -13,6 +27,12 @@ export interface Source {
   handle: string;
   type: SourceType;
   permalink: string;
+  membersCount: number;
+  description?: string;
+  members?: Connection<SourceMember>;
+  currentMember?: SourceMember;
+  owners?: string[];
+  moderators?: string[];
```

Comment:
Not sure if this is right now?
A non squad source would never have owners and moderators so why would we add it there?

A squad is technically a "extended source"

Reply:
Oh, good point. Though technically at some point it might happen (to manage the source by some employee from the source), but yes, it is not in our roadmap. Let me move these two.

Reply:
Yes that makes sense it could take a Squad or Source possibly?
And in case of a type of source being squad then run those extra fields?

Reply:
I just thought we can probably keep it `Source | Squad` and just typecast to Squad whenever necessary. Like only when checking some properties.

Reply:
Yes exactly, same thoughts 🙏

Reply:
Maybe in the long term unify through different queries so that proper types are returned. But `Source | Squad` is ok since most of squad properties are optional and we need to have checks even now.